pkg/pbq/store/wal/bootstrap.go

```diff
@@ -0,0 +1,198 @@
+package wal
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/gob"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/numaproj/numaflow/pkg/isb"
+	"github.com/numaproj/numaflow/pkg/pbq/partition"
+)
+
+var location *time.Location
+
+func init() {
```

Comment:
I think we should remove this. We just need to make sure the test data we provide is in UTC.

Reply:
i think we can remove `location` altogether.

Reply:
it might be complicated because we read from disk and build time from epoch. let's keep location.